### PR TITLE
feat(topics): optimize the example and error msg in set-retention

### PR DESCRIPTION
**Pull request by ericsyh**
### Motivation

The example command of `set-retention` for topics in pulsarctl missing the flag of `--time` and `--size`. And error msg should reminder to change the configs of `systemTopicEnabled` and `topicLevelPoliciesEnabled` in brokers to enable the topic level policy.

### CODE REVIEW

File: pkg/ctl/topic/set_retention.go [hunk 1]
```diff
@@ -27,12 +27,12 @@ import (
 func SetRetentionCmd(vc *cmdutils.VerbCmd) {
 	desc := cmdutils.LongDescription{}
 	desc.CommandUsedFor = "Set the retention policy for a topic"
-	desc.CommandPermission = "This command requires tenant admin permissions."
+	desc.CommandPermission = "This command requires tenant admin permissions and topic-level policy should be enabled in brokers"
```
Comment (zymap) [inline pkg/ctl/topic/set_retention.go:30]: I think the CommandPermission is more about the auth related things. The topic-level policy is a configuration of the broker. User should be aware of that when they setup cluster.

Comment (ericsyh) [reply pkg/ctl/topic/set_retention.go:30]: Sounds reasonnable.  I added some hint mesage for the err code so if users miss to setup the config and running the topic set-retention command, they will get enough info to update the config.

File: pkg/ctl/topic/set_retention.go [hunk 2]
```diff
@@ -27,12 +27,12 @@ import (
 func SetRetentionCmd(vc *cmdutils.VerbCmd) {
 	desc := cmdutils.LongDescription{}
 	desc.CommandUsedFor = "Set the retention policy for a topic"
-	desc.CommandPermission = "This command requires tenant admin permissions."
+	desc.CommandPermission = "This command requires tenant admin permissions"
```
Comment (zymap) [inline pkg/ctl/topic/set_retention.go:30]:
```suggestion
	desc.CommandPermission = "This command requires tenant admin permissions."
```

File: pkg/ctl/topic/set_retention.go [hunk 3]
```diff
@@ -58,7 +58,12 @@ func SetRetentionCmd(vc *cmdutils.VerbCmd) {
 		Out:  "[✖]  code: 404 reason: Namespace (tenant/namespace) does not exist",
 	}
 
-	out = append(out, successOut, noTopicName, tenantNotExistError, nsNotExistError)
+	topicLevelPolicyNotEnabledError := cmdutils.Output{
+		Desc: "topic-level policy is not enabled",
+		Out:  "[✖]  code: 405 reason: Topic level policy is disabled, please enable the topic level policy in brokers by configs of systemTopicEnabled and topicLevelPoliciesEnabled",
```
Comment (zymap) [inline pkg/ctl/topic/set_retention.go:63]: Please address the style error

Comment (ericsyh) [reply pkg/ctl/topic/set_retention.go]: I have to change the limit of line-length in golangci-lint to pass the ci...

File: golangci.yml [hunk 4]
```diff
@@ -1,6 +1,10 @@
 run:
   deadline: 6m
 
+linters-settings:
```
Comment (zymap) [inline golangci.yml:4]: Why need this? In most case we should make changes to our code, not the lint configuration

Comment (ericsyh) [reply golangci.yml:4]: The default line-length set of lint is 120 characters for one code line, while after my change of the TopicLevelPolicyNotEnabledError msg `ut: "[✖] code: 405 reason: Topic level policy is disabled, please enable broker configs of systemTopicEnabled and topicLevelPoliciesEnabled",` is almost 145 characters. 
If we need to keep the 120 characters line-length set, i think i need remove the config names in TopicLevelPolicyNotEnabledError.

Comment (zymap) [reply golangci.yml:4]: you can split the strings into two lines.  for example:
```
"[✖] code: 405 reason: Topic level policy is disabled, please enable broker "
+ "configs of systemTopicEnabled and topicLevelPoliciesEnabled"
```

Comment (ericsyh) [reply golangci.yml:4]: OK, I see.

Comment (ericsyh) [reply golangci.yml]: Upated in https://github.com/streamnative/pulsarctl/pull/439/commits/ef2b05fef7a402f415d5ef4d94bbe3dbddfc3689

**Review by zymap — APPROVED** [review]

**Pull request merged**